dts/bindings/pinctrl/nxp,mcux-rt-pinctrl.yaml

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2021 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  The node has the 'pinctrl' node label set in MCUX RT SoC's devicetree. These
+  nodes can be autogenerated using the MCUXpresso config tools combined with
+  the rt_dts_gen.py script in NXP's HAL.
+  Each pinctrl node simply selects the pinmux setting by default, sets all
+  configuration values to reset values for the SOC register. Individual
+  overrides to pinctrl nodes can be applied at the board level.
+
+  for example, here is an override for the GPIO_AD_B0_12 pad:
+
+  &IOMUXC_GPIO_AD_B0_12_LPI2C4_SCL {
+      bias-pull-up = <2>;
+      drive-strength = <7>;
+      nxp,speed = <3>;
+      slew-rate = <1>;
+  }
+
+  This will select GPIO_AD_B0_12 for use as SCL on LPI2C4, and enable
```

Comment:
Will it? It naively looks to me like this is configuring the behavior if that GPIO is used as SCL, but it doesn't seem to be actually selecting that pin for this purpose. Is this line correct? Just checking.